community/kernel/src/test/java/org/neo4j/test/assertion/Assert.java

```diff
@@ -41,6 +41,23 @@ private Assert()
     {
     }
 
+    public static <E extends Exception> void assertException( ThrowingAction<E> f, Class typeOfException ) throws
```

Comment:
isn't it copy paste mostly of another assertException method? why not reused that?